hugo/data/you_draw_it_karlsruhe.json

```diff
@@ -0,0 +1,193 @@
+{
+  "chart_id": "you-draw-it-paris",
+  "start_year_chart": 2007,
+  "end_year_chart": 2050,
+  "start_year_draw": 2019,
+  "unit": "tausend Tonnen",
+  "min_value": 0,
+  "max_value": 4000,
+  "value_label_ticks": 6,
+  "year_label_ticks": 8,
+  "values": [
+    {
+      "year": 2007,
+      "value": 3134.42
+    },
+    {
+      "year": 2008,
+      "value": 3018.888
+    },
+    {
+      "year": 2009,
+      "value": 2591.406
+    },
+    {
+      "year": 2010,
+      "value": 2700.356
+    },
+    {
+      "year": 2011,
+      "value": 2629.205
+    },
+    {
+      "year": 2012,
+      "value": 2714.637
+    },
+    {
+      "year": 2013,
+      "value": 2658.536
+    },
+    {
+      "year": 2014,
+      "value": 2596.305
+    },
+    {
+      "year": 2015,
+      "value": 2558.452
+    },
+    {
```

Comment:
Das Jahr 2015 ist hier doppelt.

Reply:
Mit der neuen you-draw-it Version #46 wird diese Datei automatisch geschrieben. Werde diesen PR gleich aktualisieren, dann sollte das hier erledigt sein.